### PR TITLE
Completed t04

### PR DESCRIPTION
**Pull request by Ajay07Metamation**

### CODE REVIEW

File: Training/Program.cs [hunk 1]
```diff
@@ -1,7 +1,43 @@
...
-        static void Main (string[] args) {
-            Console.WriteLine ("Hello, World!");
-        }
-    }
+﻿using System.Runtime;
+using static System.Console;
+var key = ConsoleKey.Y;
+while (key == ConsoleKey.Y) {
+   Write ("Enter the first number : ");
+   int.TryParse (Console.ReadLine (), out int num1);
```
Comment (rajeevramanTMM) [inline Training/Program.cs:6]: Remove unused console methods and using statements.

File: Training/Program.cs [hunk 2]
```diff
@@ -1,7 +1,43 @@
...
+         break;
+      default:
+         WriteLine ("Enter a valid key");
+         break;
+   }
+   WriteLine ("Do you want to convert again");
+   key = ReadKey ().Key;
+   WriteLine ();
+}
+int lcm (int num1, int num2) {
```
Comment (rajeevramanTMM) [inline Training/Program.cs:27]: Provide a valid method name and rewrite this expression using lambda operator.

File: Training/Program.cs [hunk 3]
```diff
@@ -1,7 +1,43 @@
...
+   int lcm = ((num1 * num2) / gcd (num1, num2));
+   return lcm;
+}
+int gcd (int num1, int num2) {
```
Comment (rajeevramanTMM) [inline Training/Program.cs:31]: Valid function name

File: Training/Program.cs [hunk 4]
```diff
@@ -1,7 +1,43 @@
...
+   int gcd = 0;
```
Comment (rajeevramanTMM) [inline Training/Program.cs:32]: Fix this negative scenario exception.
image

**Review by rajeevramanTMM — CHANGES_REQUESTED** [review]

File: Training/Program.cs (hunk 4 above)
Comment (rajeevramanTMM) [reply Training/Program.cs:30]: Please fix this error.
image

Comment (Ajay07Metamation) [reply Training/Program.cs:29]: All changes done in latest commit

**Pull request closed without merging**